include/zephyr/linker/thread-local-storage.ld

```diff
@@ -33,8 +33,4 @@
 	PROVIDE(__tbss_end = __tbss_start + __tbss_size);
 	PROVIDE(__tbss_align = ALIGNOF(tbss));
 
-	PROVIDE(__tls_start = __tdata_start);
-	PROVIDE(__tls_end = __tbss_end);
-	PROVIDE(__tls_size = __tbss_end - __tdata_start);
-
```

Comment:
> The linker variables __tls_start, __tls_end and __tls_size are not used in the zephyr project

Not true from what I'm aware, @keith-packard ?

Reply:
I just used grep for official Zephyr sources and missed picolibc completely. Seems it ist used there, so I think just reject this patch requests.

Reply:
Picolibc's default linker scripts and TLS helper functions use these symbols, but Zephyr doesn't use any of those. I think it's safe to remove these.

Reply:
@nordicjm does @keith-packard 's answer change your review?

Reply:
dismissed